pkg/ccl/changefeedccl/sink_kafka_v2.go

```diff
@@ -419,14 +424,18 @@ func buildKgoConfig(
 			}
 			tlsCfg.Certificates = []tls.Certificate{cert}
 		}
-		opts = append(opts, kgo.DialTLSConfig(tlsCfg))
+
+		dialer := &net.Dialer{Timeout: 10 * time.Second}
```

Comment:
this looks ok but pls add tests for both of these cases

Reply:
I might need some help with this. I think this code is only tested if we are connecting to a "real" endpoint.  I wasn't clear if the unit tests did this, although I do see `kafka://localhost:9092` in the tests, so they might... If we can't unit test this, is there a roachtest that sets this up that we could check the stat on?

Reply:
yeah the unit tests don't connect to real kafka. the roachtests do. but we also have a test in sink_kafka_v2_test.go that tests options being set. ideally you'd add it to both of these but given the time constraints maybe just the latter.

Reply:
I pulled the kafka changes to a separate PR, so this comment is stale, but I will look at ways to test on that PR.